decentraland/unity-renderer
# feat: add transaction dialog to accept/reject web3 operations

## What does this PR change?

Fix #1690

## How to test the changes?

1. Go to: https://play.decentraland.zone/?NETWORK=ropsten&position=133,-143&kernel-branch=feat/renderer-web3-notify&renderer-branch=feat/notify-web3
2. Log in with WALLET CONNECT
3. Press the sign/pay box
4. See if the dialog appears

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
